code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -341,6 +341,30 @@
 		else
 			icon_state = icon_opened
 
+/obj/structure/closet/secure_closet/csi

```

Comment:
You should also remove any forensics equipment from the detective locker, as I believe Bay merely merged the two together.